### DISCUSSION TITLE
Received kill signal but jobs successfull for the batch system

### DISCUSSION THREAD

**Discussion by arrabito** [DIRACGrid/DIRAC#6366] [Q&A] [answered]
Dear all,

I've noticed that some jobs fail with MinorStatus='Received Kill signal' however site admins find that the job is successfull from the batch system point of view. This behavior happens for a small fraction of identical jobs, where the successfull ones complete in several hours, while the failed ones get killed after a few seconds.

```
In the pilot log I find these lines (the full log attached):
2022-09-09 09:53:46 UTC dirac-jobexec/Script INFO: Command is: /home/execute/dir_279190/DIRAC_eaeUAupilot/19427947/dirac_prod6_run --start_run 0 --run 00000004 --with-full-moon --with-sct  Paranal gamma-diffuse North 20
Received control signal
Received Kill signal, stopping job via control signal
Sending kill signal to application PID 233
Subprocess.killChild() returned:9 
EXECUTION_RESULT[CPU] after Execution 
```
Does any one have an explanation of this behavior?
Thank you

[19427947_pilot_PIC.log](https://github.com/DIRACGrid/DIRAC/files/9575733/19427947_pilot_PIC.log)

**Comment by atsareg** [comment, marked as answer]
The log messages suggest that the job was killed by the DIRAC user, e.g. with dirac-wms-job-kill or Kill button in the Web portal. The kill instruction was sent back to the job via the heartbeat mechanism. Please, check that this is not part of your production management actions or may be a human (faulty) intervention.
